tests/unit/amo/components/TestUserProfilePictureEdit.js

```diff
@@ -0,0 +1,133 @@
+import * as React from 'react';
+
+import UserProfileEditPicture, {
+  UserProfileEditPictureBase,
+} from 'amo/components/UserProfileEditPicture';
+import { getCurrentUser } from 'amo/reducers/users';
+import UserAvatar from 'ui/components/UserAvatar';
+import {
+  dispatchSignInActions,
+} from 'tests/unit/amo/helpers';
+import {
+  fakeI18n,
+  shallowUntilTarget,
+} from 'tests/unit/helpers';
+
+
+describe(__filename, () => {
+  const render = ({ i18n = fakeI18n(), ...props } = {}) => {
+    return shallowUntilTarget(
+      <UserProfileEditPicture i18n={i18n} name="input-name" {...props} />,
+      UserProfileEditPictureBase,
+    );
+  };
+
+  it('renders itself', () => {
+    const name = 'some-input-name';
+    const root = render({ name });
+
+    expect(root.find('.UserProfileEditPicture')).toHaveLength(1);
+    expect(root.find('.UserProfileEdit--label'))
+      .toHaveText('Profile photo');
+    expect(root.find('.UserProfileEditPicture-file')).toHaveLength(1);
+    expect(root.find('.UserProfileEditPicture-file-input'))
+      .toHaveProp('name', name);
+    expect(root.find('.UserProfileEditPicture-delete-button')).toHaveLength(0);
```

Comment:
Further to my comment above, it seems like we're relying on a non-explicit default state for the component in these tests. For example here we expect there to be no delete button, but that's only because of a specific state, which we didn't set up in the test. It seems like it would be better to set up the component state in the test (by passing explicit props).